src/find.js

```diff
@@ -1,99 +0,0 @@
-const _ = require('underscore');
-
-const aggregate = require('./aggregate');
-const config = require('./config');
-const { prepareResponse, generateSort, generateCursorQuery } = require('./utils/query');
-const sanitizeParams = require('./utils/sanitizeParams');
-
-const COLLECTION_METHODS = {
-  FIND: 'find',
-  FIND_AS_CURSOR: 'findAsCursor',
-};
-
-/**
- * Performs a find() query on a passed-in Mongo collection, using criteria you specify. The results
- * are ordered by the paginatedField.
- *
- * @param {MongoCollection} collection A collection object returned from the MongoDB library's
- *    or the mongoist package's `db.collection(<collectionName>)` method.
- * @param {Object} params
- *    -query {Object} The find query.
- *    -limit {Number} The page size. Must be between 1 and `config.MAX_LIMIT`.
- *    -fields {Object} Fields to query in the Mongo object format, e.g. {_id: 1, timestamp :1}.
- *      The default is to query all fields.
- *    -paginatedField {String} The field name to query the range for. The field must be:
- *        1. Orderable. We must sort by this value. If duplicate values for paginatedField field
- *          exist, the results will be secondarily ordered by the _id.
- *        2. Indexed. For large collections, this should be indexed for query performance.
- *        3. Immutable. If the value changes between paged queries, it could appear twice.
-          4. Consistent. All values (except undefined and null values) must be of the same type.
- *      The default is to use the Mongo built-in '_id' field, which satisfies the above criteria.
- *      The only reason to NOT use the Mongo _id field is if you chose to implement your own ids.
- *    -sortAscending {boolean} Whether to sort in ascending order by the `paginatedField`.
- *    -sortCaseInsensitive {boolean} Whether to ignore case when sorting, in which case `paginatedField`
- *      must be a string property.
- *    -next {String} The value to start querying the page.
- *    -previous {String} The value to start querying previous page.
- *    -after {String} The _id to start querying the page.
- *    -before {String} The _id to start querying previous page.
- *    -hint {String} An optional index hint to provide to the mongo query
- *    -collation {Object} An optional collation to provide to the mongo query. E.g. { locale: 'en', strength: 2 }. When null, disables the global collation.
- */
```

Comment:
@adborroto I see a few files where we deleted the comments like this one. Why

Reply:
I will bring them back!